components/camel-wmq/pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.apache-extras.camel-extra</groupId>
         <artifactId>components</artifactId>
-        <version>2.18.0-SNAPSHOT</version>
+        <version>2.18-SNAPSHOT</version>
```

Comment:
By the way, I forgot in the previous review round. The version should be `2.19-SNAPSHOT` to be part of the reactor.

I will do the fix for you.

Reply:
Oh you fixed on the branch :( I cherry-pick on master then.